src/app.tsx

```diff
@@ -136,9 +135,12 @@ export function App(props: AppProps) {
       ) : null}
 
       <div className="relative z-10 flex flex-col bg-white">
-        <UtilityBar showSettings={!hasConfigUrl} />
+        <UtilityBar
+          isDoc={navItemFromUrl?.doc ?? false}
```

Comment:
I think technically this should be `activeNavItem` 

We're computing `activeNavItem` from `navItemFromUrl` up on line 97.

I don't super love how all of that stuff works. 

It might make sense to put all the logic for determining & setting the active nav item in a store? 

(See other comment about stores)

Reply:
Good catch. Both seem to work the same, but `activeNavItem` makes more sense in this context.

I don't love this either to be honest. Getting a more seasoned JS dev in here who is more familiar with stores could clean this up I imagine. I'm going to make a ticket about this, and I'll reference back to this area being on possible place for refactor/improvement.